admin/server.js

```diff
@@ -12,43 +17,45 @@ var http = require('http');
 var express = require('express');
 var bodyParser = require('body-parser');
 var cookieParser = require('cookie-parser');
-var session = require('express-session')
-var logger = require('morgan');
+var session = require('express-session');
 var xtend = require('xtend');
-var request = require('request');
 var urlJoin = require('url-join');
 var exec = require('child_process').exec;
 var app = express();
 var freeport = require('freeport');
-var multipart = require('connect-multiparty');
 var test_config = require('./test_config');
 var Users = require('../lib/users');
 
 app.set('views', __dirname + '/views');
 app.set('view engine', 'ejs');
 app.use(express.static(__dirname + '/public'));
-app.use(bodyParser.urlencoded({extended:true}));
+app.use(bodyParser.urlencoded({ extended: true }));
 app.use(cookieParser());
-app.use(session({
-  secret: 'sojo sut ed oterces le'
-}));
+app.use(
+  session({
+    secret: 'sojo sut ed oterces le',
```

Comment:
Per the [Express session middleware](https://expressjs.com/en/resources/middleware/session.html) docs regarding `secret`:
> The secret itself should be not easily parsed by a human and would best be a random set of characters. A best practice may include: The use of environment variables to store the secret, ensuring the secret itself does not exist in your repository.

Perhaps we should consider using an environment variable here which would also make it easier for maintenance in case we'd like to change it in the future.

Reply:
I am not super familiar with the repercussions of this. Going to leave this for an N+1